core/src/epicli/data/common/ansible/playbooks/roles/postgres_exporter/tasks/main.yml

```diff
@@ -0,0 +1,99 @@
+---
+- name: Install pip3 [Debian]
+  apt:
+    name:
+      - libpq5
+      - python3-psycopg2
+      - python3-jmespath
+    state: present
+    install_recommends: no
```

Comment:
```suggestion
    install_recommends: false
```

Reply:
https://docs.ansible.com/ansible/2.3/apt_module.html there is no true / false in documentation